src/OrderBook.ts

```diff
@@ -255,18 +264,19 @@ export default class OrderBook {
     return topBuy ? topBuy[EnumLevelProperty.price] : null;
   }
 
-  /**
-   * get current bid/ask spread percentage
-   * @param {number} [n=0] offset from centre of book
-   * @returns {number} percentage spread between best bid & ask
-   */
-  public getSpreadPercent(n = 0): number | null {
-    const ask = this.getBestAsk(n);
-    const bid = this.getBestBid(n);
-
-    if (!bid || !ask) {
-      return null;
-    }
-    return (1 - bid / ask) * 100;
-  }
+  /**  
+   * get current bid/ask spread in basis points  
+   * @param {number} [n=0] offset from centre of book  
+   * @returns {number} spread between best bid & ask in basis points  
+   */  
+  public getSpreadBasisPoints(n = 0): number | null {  
+    const ask = this.getBestAsk(n);  
+    const bid = this.getBestBid(n);  
+    
+    if (!bid || !ask) {  
+      return null;  
+    }  
+    // calculate spread in basis points  
+    return ((1 - bid / ask) * 100) * 100;  
+  }  
```

Comment:
Thanks for adding this! Since it's a different method & measurement all together, can you also add back the `getSpreadPercent` method that you removed? This way, it's not a breaking change.